lhotse/utils.py

```diff
@@ -171,9 +180,28 @@ def split_sequence(seq: Sequence[Any], num_splits: int, shuffle: bool = False) -
         raise ValueError(f"Cannot split iterable into more chunks ({num_splits}) than its number of items {num_items}")
     if shuffle:
         random.shuffle(seq)
-    chunk_size = int(ceil(num_items / num_splits))
-    split_indices = [(i * chunk_size, min(num_items, (i + 1) * chunk_size)) for i in range(num_splits)]
-    return [seq[begin: end] for begin, end in split_indices]
+    chunk_size = int(math.floor(num_items / num_splits))
```

Comment:
```python
chunk_size = num_items // num_splits
```
is much clearer.

By the way, the return type of `math.floor` is an `int`. It's redundant to use an extra
conversion here.